Dockerfile

```diff
@@ -49,3 +49,5 @@ RUN sh /tmp/update_selinon.sh
 RUN cd /tmp/install_deps/ && /tmp/install_deps/apply_patches.sh
 
 RUN yum install -y npm && npm install -g semver-ranger
+
+RUN pip3 uninstall -y protobuf && pip3 install packaging appdirs && pip3 install --upgrade --no-binary :all: protobuf==3.3.0
```

Comment:
We encountered a very weird Import Error with Google Cloud lib. Even if the lib is installed properly and we can see manually that it is available in import-path, but still the program throws Import Error for "protobuf" !

I discussed it in matter-most and Saket/Slavek shared the above work-around.
Reference: https://chat.openshift.io/rh-devtools/pl/5e6sbcsyz3fuxfw1zs4n97hwww